x-pack/plugin/sql/qa/src/main/resources/datetime.csv-spec

```diff
@@ -726,9 +788,9 @@ M       | 1996-11-05 00:00:00.000Z
 castedDateTimeWithGroupBy1
 SELECT CONVERT(birth_date, DOUBLE) AS date FROM test_emp GROUP BY date ORDER BY date LIMIT 10;
 
-    date:d      
+    date:d
```

Comment:
Here and below: please revert the formatting changes, It's nice to remove the whitespaces but can make it confusing in the git history of the file.